api/routes/cartRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+
+const cartController = require("../controllers/cartController");
+
+const router = express.Router();
+
+router.get("/cart/:userId", cartController.getCarts);
+
+router.post("/cart", cartController.postCart);
+
+router.delete("/cart/:cartId", cartController.deleteCart);
```

Comment:
해당 부분은 장바구니 URI로 보입니다. 먼저, cartRouter에서 분기되기 전에 `index.js`에서 
```
// index.js
const express = require("express");

const router = express.Router();

const cartRouter = require("./cartRouter");

router.use("/cart", cartRouter.router);

module.exports = router;
```
위 처럼 분기하고 
해당 URI는 
```suggestion
router.get("/:userId", cartController.getCarts);
router.post("", cartController.postCart);
router.delete("/:cartId", cartController.deleteCart);
```
위 처럼 router로 분기할 수 있도록 수정해 주세요. `routes/index.js`로 기능별로 분기가 되기 때문입니다. 그렇기에 index.js에서 우선적으로 `/cart` 로 하여금 분기하는 과정이 필요합니다.